velox/functions/lib/MapConcat.cpp

```diff
@@ -109,6 +113,11 @@ class MapConcatFunction : public exec::VectorFunction {
     // Check for duplicate keys
     SelectivityVector uniqueKeys(offset);
     vector_size_t duplicateCnt = 0;
+    const auto throwExceptionOnDuplicateMapKeys =
+        context.execCtx()
+            ->queryCtx()
+            ->queryConfig()
```

Comment:
Can you add a null check here? Some use case does not have query config set

Reply:
`queryConfig()` returns `const QueryConfig&`, and if the values in it is empty `sparkThrowExceptionOnDuplicateMapKeys()` would return the default value 'false' as key cannot be found in the map. Do you think it makes sense? Thanks.

Reply:
Sorry it's the `QueryCtx` that can be null

Reply:
I understand. Updated as below, would you help take another look? Thanks!

```
    bool throwExceptionOnDuplicateMapKeys = false;
    if (auto* ctx = context.execCtx()->queryCtx()) {
      throwExceptionOnDuplicateMapKeys =
          ctx->queryConfig().sparkThrowExceptionOnDuplicateMapKeys();
    }
```